### DISCUSSION TITLE
OpenID login FAIL

### DISCUSSION THREAD

**Discussion by barisorhun** [danny-avila/LibreChat#4077] [Troubleshooting]
Since updating LibreChat to the most recent version, OpenID login which was previously setup is not working anymore. Checked all my secrets for their validity and all is well. Below is what I see in the console when I attempt to log in:

"Token is not present, user is not authenticated"

Reminder: None of my secrets changed any time soon. Reverting back to the older versions **does** let me log in with OpenID fine. I suspect I'm missing some sort of configuration with respect to the authentication changes that was made to repo. I'd appreciate any sort of help for this. Thank you for your time.

EDIT: Tried the openid ( Entra ) login with re-doing the steps in https://www.librechat.ai/docs/configuration/authentication/OAuth2-OIDC/azure
Issue persists with brandh new registered app/secrets.

**Comment by danny-avila** [comment]
I followed the exact instructions for Microsoft Entra from the guide you linked, and it worked for me.

See video: 

https://github.com/user-attachments/assets/051a930b-4ef2-4b0b-b48d-dd1f12f150a6



There have been no changes to OpenID nor any authentication changes that should be affecting you.

**Comment by danny-avila** [comment]
It would be more helpful to share backend logs for what's going on, in particular, error logs that include stack traces.

If you are using docker, they are saved in the `./logs` directory at project root.

Reproduce the issue and then check the latest error log (file starting "error" in name, i.e. `logs/error-2024-09-21.log`)
